egs/tedlium/s5/local/chain/run_tdnn_cleanup.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+#
```

Comment:
also with this script, see if you can merge it into the run_tdnn.sh.  
We may make cleanup the default recipe for these setups.
Please get rid of any options that you think are not helpful.